packages/next/build/swc/src/styled_jsx/mod.rs

```diff
@@ -22,753 +22,754 @@ mod transform_css;
 mod utils;
 
 pub fn styled_jsx() -> impl Fold {
-  StyledJSXTransformer::default()
+    StyledJSXTransformer::default()
 }
 
 #[derive(Default)]
 struct StyledJSXTransformer {
-  styles: Vec<JSXStyle>,
-  static_class_name: Option<String>,
-  class_name: Option<Expr>,
-  file_has_styled_jsx: bool,
-  has_styled_jsx: bool,
-  bindings: AHashSet<Id>,
-  nearest_scope_bindings: AHashSet<Id>,
-  style_import_name: Option<String>,
-  external_bindings: Vec<Id>,
-  file_has_css_resolve: bool,
-  external_hash: Option<String>,
-  add_hash: Option<(String, String)>,
-  add_default_decl: Option<(String, Expr)>,
-  evaluator: Option<Evaluator>,
+    styles: Vec<JSXStyle>,
+    static_class_name: Option<String>,
+    class_name: Option<Expr>,
+    file_has_styled_jsx: bool,
+    has_styled_jsx: bool,
+    bindings: AHashSet<Id>,
+    nearest_scope_bindings: AHashSet<Id>,
+    style_import_name: Option<String>,
+    external_bindings: Vec<Id>,
+    file_has_css_resolve: bool,
+    external_hash: Option<String>,
+    add_hash: Option<(String, String)>,
+    add_default_decl: Option<(String, Expr)>,
+    evaluator: Option<Evaluator>,
```

Comment:
Are these whitespace changes intentional?

Reply:
It's a result of `rustfmt`. 

I'm not sure, but running `rustfmt` for all files by doing `cargo fmt` on commit may worth it.